lib/model/model.go

```diff
@@ -1095,9 +1109,9 @@ func (m *Model) ResumeDevice(device protocol.DeviceID) {
 
 func (m *Model) IsPaused(device protocol.DeviceID) bool {
 	m.pmut.Lock()
-	paused := m.devicePaused[device]
-	m.pmut.Unlock()
-	return paused
+	defer m.pmut.Unlock()
+
+	return m.devicePaused[device]
```

Comment:
`defer` kills potential inlining by the compiler (or so I've been told), and given this is small, it looks like a good candidate for inlining.